packages/ui/src/components/va-carousel/VaCarousel.stories.ts

```diff
@@ -1,12 +1,224 @@
+  `,
+})
+
+export const Arrows = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems(), current: 0 }),
+  template: `
+    [true]
+    <VaCarousel :items="items" :indicators="false" v-model="current"/>
+    [false]
+    <VaCarousel :items="items" :indicators="false" :arrows="false" v-model="current"/>
+  `,
+})
+
+Arrows.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnArrowClick = canvas.getByRole('button', { name: 'go next slide' }) as HTMLElement
+
+  await step('Should head to slide 2 by clicking on arrow', async () => {
+    await userEvent.click(NavigatesOnArrowClick)
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+}
+
+export const Indicators = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems(), current: 0 }),
+  template: `
+    [true]
+    <VaCarousel :items="items" :arrows="false" v-model="current"/>
+    [false]
+    <VaCarousel :items="items" :arrows="false" :indicators="false" v-model="current"/>
+  `,
+})
+
+Indicators.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnPageClick = canvas.getByRole('button', { name: 'go slide 2' }) as HTMLElement
+
+  await step('Should head to slide 2 by clicking on indicator', async () => {
+    await userEvent.click(NavigatesOnPageClick)
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+}
+
+export const IndicatorsTrigger = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    [hover]
+    <VaCarousel :items="items" :arrows="false" indicator-trigger="hover"/>
+    [default]
+    <VaCarousel :items="items" :arrows="false" />
+  `,
+})
+
+IndicatorsTrigger.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnPageClick = canvas.getAllByRole('button', { name: 'go slide 2' }) as HTMLElement[]
+
+  await step('Should head to slide 2 by hover on indicator', async () => {
+    await userEvent.hover(NavigatesOnPageClick[0])
+    await userEvent.unhover(NavigatesOnPageClick[0])
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+
+  await step('Should remain on slide 1 by hover on indicator', async () => {
+    await userEvent.hover(NavigatesOnPageClick[1])
+    await userEvent.unhover(NavigatesOnPageClick[1])
+    expect(slides[3].getAttribute('aria-current')).toEqual('true')
+  })
 }
 
-export const Default = defineComponent({
+export const Infinite = () => ({
   components: { VaCarousel },
-  template: '<VaCarousel/>',
+  data: () => ({ items: getItems() }),
+  template: `
+    [infinite]
+    <VaCarousel :items="items" infinite />
+    [default]
```

Comment:
please use actual values here: `true` and `false`. 

`infinite` in template actually means `infinite=true`, and no infinite at all means `infininte=false`. Even if doesn't look so.

It would probably be for the best to use infinite=true by default. I checked vuetify and naive ui and it seems meaningful from my experience.

Reply:
Sorry for not being clear.

I meant
* `[infinite]` replace with `[true]`
*  and `[default]` replace with `[false]` 

Template overall was fine. Ping me in discord if it's unclear why this is needed.

<img width="401" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/8a1d6012-950a-496a-a445-c0ca86f4240b">